src/test/java/org/jboss/set/state/ConfigDeserializationTest.java

```diff
@@ -0,0 +1,57 @@
+package org.jboss.set.state;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import io.quarkus.test.junit.QuarkusTest;
+import jakarta.inject.Inject;
+import org.jboss.config.GitHubRawUrl;
+import org.jboss.config.LotteryConfig;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+@QuarkusTest
+public class ConfigDeserializationTest {
+
+    @Inject
+    ObjectMapper objectMapper;
+
+    @Test
+    void testDeserializingSimpleConfigFile() {
+        String configFile = """
+                delay: P14D
```

Comment:
just what is P14D?

Reply:
14 days, this needs to be in format for `java.time.Duration`, used as default converter in Quarkus for time ref: https://quarkus.io/guides/config-reference#default-values